index.js

```diff
@@ -1,17 +1,50 @@
 class SortedList {
-  constructor() {}
+  constructor() {
 
-  add(item) {}
+    this.items = [];
+    this.length = this.items.length;
 
-  get(pos) {}
+  }
 
-  max() {}
+  add(item) {
 
-  min() {}
+    this.items.push(item);  
 
-  sum() {}
+    this.items.sort((a, b) => a - b);
+    
+    this.length = this.items.length;
+  }
 
-  avg() {}
-}
+  get(pos) {
+    if (this.items[pos]) return this.items[pos];
```

Comment:
Aqui vc pode retornar um ternário:

return this.items[pos] ? this.items[pos] : throw new Error('OutOfBounds');

Fica legal assim :-)